shanoir-ng-front/src/app/preclinical/importBruker/bruker-upload/bruker-upload.component.ts

```diff
@@ -82,12 +82,12 @@ export class BrukerUploadComponent {
         this.fileToUpload = file.item(0);
     	this.uploadProgress = 1;
     	this.importBrukerService.postFile(this.fileToUpload)
-        	.subscribe(res => {
```

Comment:
this was not unsubscribed, stacking indefinitely one instance of this code block at every call of the function